aws/ecs_cluster_test.go

```diff
@@ -0,0 +1,104 @@
+package aws
+
+import (
+	"testing"
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/gruntwork-io/cloud-nuke/util"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const region = "eu-west-1"
+
+// Test we can create a cluster, tag it, and then find the tag
+func TestCanTagEcsClusters(t *testing.T) {
+	t.Parallel()
+
+	awsSession, err := session.NewSession(&awsgo.Config{
+		Region: awsgo.String(region),
+	})
+	require.NoError(t, err)
+
+	cluster := createEcsFargateCluster(t, awsSession, util.UniqueID())
+	defer deleteEcsCluster(awsSession, cluster)
+
+	tagValue := time.Now().UTC().Format(time.RFC3339)
+
+	tagEcsCluster(awsSession, cluster.ClusterArn, firstSeenTagKey, tagValue)
+	require.NoError(t, err)
```

Comment:
```suggestion
	tagErr := tagEcsCluster(awsSession, cluster.ClusterArn, firstSeenTagKey, tagValue)
	require.NoError(t, tagErr)
```

Reply:
BTW, your IDE should warn you loudly about unhandled errors in Go, as they are almost always bugs. Make sure to turn that on!

Reply:
Ah yes - I'm looking to see what extension would catch such sort of an overlook, or what is the VSCode setting i need switched on. I know we chatted briefly about this, so I'll ask a question on Slack about what people use.